content/en/docs/_index.md

```diff
@@ -1,20 +1,42 @@
 
 ---
-title: "Tekton Documentation"
+title: "Welcome to Tekton"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-Tekton is a cloud native continuous integration and delivery (CI/CD)
-solution. It allows developers to build, test, and deploy across cloud
-providers and on-premise systems.
+Tekton is an open-source cloud native CI/CD (Continuous Integration and
+Delivery/Deployment) solution. It allows developers to build, test, and
+deploy across destinations using a Kubernetes cluster of their own.
 
-### What's next
+The Tekton project consists of the following components:
```

Comment:
We already had this documented very well in https://github.com/tektoncd/website/pull/168, but now I cannot find this file. What ever happened to it?

Reply:
Looks like it was moved in https://github.com/tektoncd/website/pull/214
Now it's at https://github.com/tektoncd/website/blob/2b51b90d0a4720c2c5d6a02b8c6ec8b1c8ce64ef/content/en/docs/Overview/_index.md

Reply:
Then it's probably best to link to the Overview instead of rehashing this content in slightly different wording.

Reply:
Agreed, duplication makes it harder to keep things consistent and up to date. The overview is already linked from the current landing page: 
![image](https://user-images.githubusercontent.com/2829095/112338102-6ae10380-8cb6-11eb-82f5-2ff85015afdd.png)